tasks/install.yml

```diff
@@ -0,0 +1,39 @@
+---
+
+- name: Add Erlang repository key
+  become: yes
+  apt_key:
+    url: https://packages.erlang-solutions.com/ubuntu/erlang_solutions.asc
+
+- name: Add Erlang repository
+  become: yes
+  apt_repository:
+    repo: "deb https://packages.erlang-solutions.com/ubuntu xenial contrib"
+    update_cache: yes
+
+- name: Ensure Erlang is installed
```

Comment:
This might seem kinda nitty, but I would recommend combining this package install with the rabbitmq one and turn it into a list of items. This would support the installation of additional packages if needed. This would remove two calls to update the apt cache on each repo task as well, since we can use `update_cache` on the package install task.